miso-web/src/main/webapp/scripts/sample_hot.js

```diff
@@ -914,6 +915,17 @@ Sample.hot = {
         // send it through the parser to get a sampleData array that isn't merely a reference to Sample.hot.hotTable.getSourceData()
         var sampleData = JSON.parse(JSON.parse(JSON.stringify(Sample.hot.hotTable.getSourceData())));
         
+        // Array of save functions, one for each line in the table.
+        var sampleSaveArray = new Array();
+        
```

Comment:
From what I understand, it's more idiomatic in JavaScript to use ``var sampleSaveArray = [];``